docs/src/main/sphinx/glossary.rst

```diff
@@ -0,0 +1,187 @@
+========
+Glossary
+========
+
+The glossary contains a list of key Trino terms and definitions.
+
+.. _glossCatalog:
+
+Catalog
+    Catalogs define and name a configuration for connecting to a data source,
+    allowing users to query the connected data. Each catalog's configuration
+    specifies a :ref:`connector <glossConnector>` to define which data source
+    the catalog connects to. For more information, see
+    :ref:`trino-concept-catalog`.
+
+.. _glossCA:
+
+Certificate Authority (CA)
+    A trusted organization that examines and validates organizations and their
+    proposed server URIs, and issues digital certificates verified as valid for
+    the requesting organization.
+
+.. _glossCert:
+
+Certificate
+    A public key `certificate
+    <https://en.wikipedia.org/wiki/Public_key_certificate>`_ issued by a CA,
+    sometimes abbreviated as cert, that verifies the ownership of a server's
+    keys. Certificate format is specified in the `X.509
+    <https://en.wikipedia.org/wiki/X.509>`_ standard.
+
+Cluster
+    A Trino cluster provides the resources to run queries against numerous data
+    sources. Clusters define the number of workers, the configuration for the
+    JVM runtime, configured data sources, and others aspects. For more
+    information, see :ref:`trino-concept-cluster`.
+
+.. _glossConnector:
+
+Connector
```

Comment:
Nit: could you add a definition for a Trino plugin? It would be nice to have a reference to distinguish the difference between a connector and a plugin.

Reply:
Good idea!

Reply:
Done!